jina/logging/formatter.py

```diff
@@ -6,6 +6,9 @@
 from .profile import used_memory
 from ..helper import colored
 
+if False:
```

Comment:
Just curious how is 'if False' used here

Reply:
It is used to have `flake` not complaint when the class of the `type hint` is not imported while avoid the extra import. 

You can see is a `trick` used all over the place

Reply:
Thanks!